core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -299,6 +299,7 @@ class RenderBlockingResources extends Audit {
       numericValue: wastedMs,
       numericUnit: 'millisecond',
       details,
+      metricSavings: {FCP: wastedMs, LCP: wastedMs},
```

Comment:
Using the existing `wastedMs` makes sense for now, but `wastedMs` is computed using the FCP graph. I do wonder if computing a second LCP `wastedMs` would also be useful.

This requires more investigation, so I'll open a separate issue to track this.

Reply:
ah something like what we have to do with `uses-rel-preconnect`? 

separate issue sounds good to me!

Reply:
Yeah, I imagine we'll copy the FCP implementation for LCP and see how big the difference is.